litestar/dto/config.py

```diff
@@ -10,10 +10,7 @@
 
     from litestar.dto.types import RenameStrategy
 
-__all__ = (
-    "DTOConfig",
-    "SQLAlchemyDTOConfig",
```

Comment:
While this shouldn't have been here before itself, since this is already here, should this still be included to maintain backwards compatibility? Or was there no release after this was included?

Reply:
It doesn't look like the class was actually imported into the module though, so it couldn't have been depended upon in any case.

Reply:
It was defined within this module itself.

Reply:
There hasn't been a release that we've included `SQLAlchemyDTOConfig`, so we can change this if we'd like.

Reply:
> It was defined within this module itself.

It was :open_mouth: - I thought this stuff was in `contrib` - sorry still catching up.